test/unit/org/apache/cassandra/service/accord/AccordTestUtils.java

```diff
@@ -517,12 +517,9 @@ public static void appendCommandsBlocking(AccordCommandStore commandStore, Comma
 
     public static void appendCommandsBlocking(AccordCommandStore commandStore, Command before, Command after)
     {
-        SavedCommand.SavedDiff diff = SavedCommand.diff(before, after);
-        if (diff != null)
-        {
-            Condition condition = Condition.newOneTimeCondition();
-            commandStore.appendCommands(Collections.singletonList(diff), null, condition::signal);
-            condition.awaitUninterruptibly(30, TimeUnit.SECONDS);
-        }
+        SavedCommand.Writer<TxnId> diff = new SavedCommand.DiffWriter(before, after);
```

Comment:
should we try to respect the logic in `org.apache.cassandra.service.accord.AccordSafeCommand#diff`?  Maybe make this a static function so this code and that function can both use?  if `before=null, after=Uninitialised` its a no-op...

Reply:
made that change in https://github.com/ifesdjeen/cassandra/pull/2